templates/expo-template-blank-typescript/gitignore

```diff
@@ -35,3 +35,7 @@ yarn-error.*
 
 # typescript
 *.tsbuildinfo
+
+# generated native folders
+ios/
```

Comment:
this should be `/ios` and `/android` -- otherwise we also ignore native directories in local expo modules, so `modules/my-module/android/build.gradle` etc would all be ignored

Reply:
Ah of course, good spot thank you. This is actually another common source of confusion - we've had several people on discord whose expo modules didn't work on EAS because the native folders were ignored.